vdev/src/commands/release/prepare.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::Result;
+
+use crate::app;
+
+/// Update Kubernetes manifests from latest stable release and create a new Cue file for the new
+/// release
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        // Dummy arguments to satisfy `impl` trait bound
```

Comment:
That's a little awkward, how much effort would it be to allow that to be optional?